platform/ext/target/nordic_nrf/nrf5340/target_cfg.h

```diff
@@ -38,6 +38,14 @@ struct memory_region_limits {
 #endif /* BL2 */
 };
 
+/**
+ * \brief Holds the data necessary to do isolation for a specific peripheral.
+ */
+struct tfm_spm_partition_platform_data_t
```

Comment:
Since you introduced code that uses it, I think you should add `periph_start` and `periph_limit` also.

Reply:
why?
these are needed to configure isolation using ARM's PPC, but Nordic does not need them. Only the base address is needed by the SPU.

Reply:
They are used in lines 68 and 69 in the code you added.

Reply:
Oh oh, right, missed that - i need them yes, thanks

Reply:
[the SPU _does_ need only the base address, but the MPU needs the limits.]

Reply:
fixed, thanks